packages/i/irrXML/xmake.lua

```diff
@@ -0,0 +1,16 @@
+package("irrXML")
+
+    set_homepage("https://sourceforge.net/projects/irrlicht/")
+    set_description("High speed and easy-to-use XML Parser for C++")
+
+    set_urls("https://deac-fra.dl.sourceforge.net/project/irrlicht/irrXML%20SDK/$(version)/irrxml-$(version).zip")
+    add_versions("1.2", "9b4f80639b2dee3caddbf75862389de684747df27bea7d25f96c7330606d7079")
+    add_patches("1.2", path.join(os.scriptdir(), "patches", "1.2", "add-xmake.patch"), "a7eb6abd96a9c4b98824c2101378022cb905d8ed64b9a7fc5badc13c3188438c")
+
+    on_install(function (package)
```

Comment:
If xmake.lua is small and not much different from the version, it is recommended to write it directly to the current source directory.

```lua
io.writefile("xmake.lua", [[
add_rules("mode.debug", "mode.release")
target("irrXML")
    set_kind("static")
    add_headerfiles("src/*.h")
    add_files("src/*.cpp")]]

Reply:
In this case, writing directly to xmake.lua will be more convenient for maintenance than adding patches.